examples/clickpipe/kafka_azure_eventhub/provider.tf

```diff
@@ -8,8 +8,16 @@ terraform {
   }
 }
 
+variable "organization_id" {}
+variable "token_key" {}
+variable "token_secret" {}
+variable "api_url" {
+  default = "https://api.clickhouse.cloud/v1"
```

Comment:
this is already the default on the client API side, why are you defaulting it again?